sdk/src/connections/usb/windows/usb_depth_sensor_windows.cpp

```diff
@@ -704,6 +728,18 @@ aditof::Status UsbDepthSensor::writeAfeRegisters(const uint16_t *address,
 
     ExUnitHandle handle;
 
+    if (address == nullptr) {
+        LOG(ERROR) << "Received AfeRegisters address null pointer";
+        return Status::INVALID_ARGUMENT;
+    }
+
+    if (data == nullptr) {
+        LOG(ERROR) << "Received AfeRegisters data null pointer";
+        return Status::INVALID_ARGUMENT;
+    }
+
+    assert(length > 0);
```

Comment:
`length` has the type `size_t` which by definition is unsigned, so this is not needed.

Reply:
In this way we will prevent the length to be set to 0.